locales/de.json

```diff
@@ -0,0 +1,21 @@
+{
+  "components.empty-view.welcome_back_title": "Willkommen zurück bei Buttercup.",
+  "components.empty-view.unlock_archive": "Entsperre ein Archiv um zu starten ({os}).",
+  "components.empty-view.welcome_title": "Willkommen bei Buttercup.",
+  "components.empty-view.welcome_caption": "Du hast bis jetzt noch keine Archive. Warum fügst du nicht eins hinzu?",
+  "components.add-archive-button.open_archive_file": "Archiv öffnen",
+  "components.add-archive-button.new_archive_file": "Archiv erstellen",
+  "components.add-archive-button.connect_cloud_sources": "Einen Cloud Services verbinden",
+  "components.add-archive-button.add_archive": "Archiv hinzufügen",
+  "components.file-manager.index.cancel": "Abbrechen",
+  "components.file-manager.index.go_back": "Zurück",
+  "components.file-manager.index.new_archive": "Neues Archiv",
+  "components.file-manager.index.open_in_buttercup": "Mit Buttercup öffnen",
+  "system.dialog.title": "Master Passwort",
+  "system.dialog.confirm": "Bestätigen",
+  "system.dialog.nevermind": "Abbrechen",
+  "system.dialog.password": "Passwort",
+  "main.menu.language": "Sprache",
```

Comment:
I really like the nested style, though I think some components should be named differently. For instance, `main.menu` alludes to there being a `main` category with a `menu` subcategory, which I don't think there is. Should this be `main-menu` perhaps? Or even `menu.main`?